README.md

```diff
@@ -144,3 +144,25 @@ Generates this output:
 █████████████████████████████
 ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
 ```
+
+## PIC generation
+
+```rust
+use qrcode::render::pic;
+use qrcode::QrCode;
+
+fn main() {
+    let code = QrCode::new(b"01234567").unwrap();
+    let image = code
+        .render()
+        .min_dimensions(1, 1)
+        .dark_color(pic::Color("#800000"))
+        .light_color(pic::Color("#ffff80"))
+        .build();
+    println!("{}", image);
+}
+```
+
+Generates PIC output that renders as follows:
```

Comment:
PIC seems like some obscure markup format unlike PNG or SVG. I suppose it is https://en.wikipedia.org/wiki/PIC_(markup_language)? Please provide a link to introduce what PIC is.

Reply:
Correct that's the PIC this PR adds support for.
Would linking to a PDF be acceptable?
I found ["PIC — A Graphics Language for Typesetting"](https://archive.org/details/pic-graphics-language/mode/2up) the best source material.

Reply:
The primary introduction should not be this PDF file, in the same way if I were to introduce SVG I wouldn't link to https://www.w3.org/TR/SVG11/. 

This PDF file is a nice addition as a "for more details" link.